app/src/features/reports/VariableReport.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useState, useEffect } from "react";
+import { Grid } from "@material-ui/core";
+import TableChart from "../charts/TableChart";
+import styles from "./Report.module.scss";
+import WithDatasets from "../../utils/WithDatasets";
+import useDatasetStore from "../../utils/useDatasetStore";
+import variableProviders, { VariableId } from "../../utils/variableProviders";
+import { DropdownVarId } from "../../utils/MadLibs";
+import { Breakdowns } from "../../utils/Breakdowns";
+import VariableProvider from "../../utils/variables/VariableProvider";
+import { USA_FIPS, Fips } from "../../utils/Fips";
+import MapNavChart from "../charts/MapNavChart";
+import Alert from "@material-ui/lab/Alert";
+import Card from "@material-ui/core/Card";
+
+const VARIABLE_DISPLAY_NAMES: Record<string, Record<string, string>> = {
+  diabetes: {
+    diabetes_count: "Diabetes Case Count",
+  },
+  copd: {
+    copd_count: "COPD Case Count",
+  },
+};
+
+function VarGeoReport(props: {
+  variable: DropdownVarId;
+  stateFips: string;
+  updateStateCallback: Function;
+  vertical?: boolean;
+}) {
+  // TODO Remove hard coded fail safe value
+  const variableId: VariableId = Object.keys(VARIABLE_DISPLAY_NAMES).includes(
+    props.variable
+  )
+    ? (Object.keys(VARIABLE_DISPLAY_NAMES[props.variable])[0] as VariableId)
+    : ("diabetes_count" as VariableId);
+  const variableDisplayName = Object.keys(VARIABLE_DISPLAY_NAMES).includes(
+    props.variable
+  )
+    ? Object.entries(VARIABLE_DISPLAY_NAMES[props.variable])[0][1]
+    : "Placeholder";
+
+  const datasetStore = useDatasetStore();
+  const variableProvider = variableProviders[variableId];
+  const requiredDatasets = VariableProvider.getUniqueDatasetIds([
+    variableProvider,
+  ]);
+
+  const [fips, setFips] = useState<Fips>(new Fips(props.stateFips));
+
+  useEffect(() => {
+    console.log(props.stateFips);
+    setFips(new Fips(props.stateFips));
+  }, [props.stateFips]);
```

Comment:
Same question as before - can this be lifted up one more level to the parent component?

Reply:
It's a little more complicated as there isn't a "county" in the MadLib dropdown menu... This is touching on what we were discussing today about being able to type in a county. I'm going to drop a TODO and explore it in another PR since this is already pretty huge

Reply:
It's complicated by the MadLib dropdown not handling the County FIPs the same way it can handle the State FIPs. This touches on what we were discussing in the meeting a couple hours ago- I'll incorporate this suggestion into that PR instead since this one is already big and I've already made a dent over there :)

Reply:
ugh github is hard. sorry for the double comment ><"